README.adoc

```diff
@@ -214,7 +214,7 @@ Use Unix-style line endings.footnote:[*BSD/Solaris/Linux/macOS users are covered
 ====
 If you're using Git you might want to add the following configuration setting to protect your project from Windows line endings creeping in:
 
-[source,bash]
+[source,shell]
```

Comment:
For consistency. I know this is the valid source. https://docs.asciidoctor.org/asciidoc/latest/verbatim/source-highlighter/#shell-vs-console

Reply:
What is worth noticing in the link:
> [for `console`, ] the syntax highlighter will parse the prompt (e.g., $) at the start of each line, then handle the remaining text using the shell language

Should we use `console` here?

Reply:
Yeah, `console` looks better here.